docs/services/storage.md

```diff
@@ -57,15 +53,6 @@ token:
       network: default # the name of the network this TMS refers to (Fabric, Orion, etc)
       channel: testchannel # the name of the network's channel this TMS refers to, if applicable
       namespace: tns # the name of the channel's namespace this TMS refers to, if applicable
-
```

Comment:
can you clarify that one defaults to the default under `persistences`?

Reply:
Yes, I have mentioned it in FSC, but I can also mention it here.